src/java.base/share/classes/java/util/regex/CharPredicates.java

```diff
@@ -170,55 +170,51 @@ static final CharPredicate WORD() {
+            case "CONTROL" -> CONTROL();
+            case "HEXDIGIT", "HEX_DIGIT" -> HEX_DIGIT();
+            case "IDEOGRAPHIC" -> IDEOGRAPHIC();
+            case "JOINCONTROL", "JOIN_CONTROL" -> JOIN_CONTROL();
+            case "LETTER" -> LETTER();
+            case "LOWERCASE" -> caseIns
                                 ? LOWERCASE().union(UPPERCASE(), TITLECASE())
                                 : LOWERCASE();
-            case "UPPER": return caseIns
+            case "NONCHARACTERCODEPOINT", "NONCHARACTER_CODE_POINT" -> NONCHARACTER_CODE_POINT();
+            case "TITLECASE" -> caseIns
+                                ? TITLECASE().union(LOWERCASE(), UPPERCASE())
+                                : TITLECASE();
+            case "PUNCTUATION" -> PUNCTUATION();
+            case "UPPERCASE" -> caseIns
                                 ? UPPERCASE().union(LOWERCASE(), TITLECASE())
                                 : UPPERCASE();
-            case "SPACE": return WHITE_SPACE();
-            case "PUNCT": return PUNCTUATION();
-            case "XDIGIT": return HEX_DIGIT();
-            case "ALNUM": return ALNUM();
-            case "CNTRL": return CONTROL();
-            case "DIGIT": return DIGIT();
-            case "BLANK": return BLANK();
-            case "GRAPH": return GRAPH();
-            case "PRINT": return PRINT();
-            default: return null;
-        }
-    }
-
-    private static CharPredicate getUnicodePredicate(String name, boolean caseIns) {
-        switch (name) {
-            case "ALPHABETIC": return ALPHABETIC();
-            case "ASSIGNED": return ASSIGNED();
-            case "CONTROL": return CONTROL();
-            case "HEXDIGIT":
-            case "HEX_DIGIT": return HEX_DIGIT();
-            case "IDEOGRAPHIC": return IDEOGRAPHIC();
-            case "JOINCONTROL":
-            case "JOIN_CONTROL": return JOIN_CONTROL();
-            case "LETTER": return LETTER();
-            case "LOWERCASE": return caseIns
-                                    ? LOWERCASE().union(UPPERCASE(), TITLECASE())
-                                    : LOWERCASE();
-            case "NONCHARACTERCODEPOINT":
-            case "NONCHARACTER_CODE_POINT": return NONCHARACTER_CODE_POINT();
-            case "TITLECASE": return caseIns
-                                    ? TITLECASE().union(LOWERCASE(), UPPERCASE())
-                                    : TITLECASE();
-            case "PUNCTUATION": return PUNCTUATION();
-            case "UPPERCASE": return caseIns
-                                    ? UPPERCASE().union(LOWERCASE(), TITLECASE())
-                                    : UPPERCASE();
-            case "WHITESPACE":
-            case "WHITE_SPACE": return WHITE_SPACE();
-            case "WORD": return WORD();
-            default: return null;
-        }
+            case "WHITESPACE", "WHITE_SPACE" -> WHITE_SPACE();
+            case "WORD" -> WORD();
+            default -> null;
+        };
```

Comment:
This file has lots of changes which are difficult to review. Maybe it should be split out of this PR.

Reply:
*sigh* GitHub diff tool is really poor and outdated. Here's how it looks in IntelliJ IDEA diff view:
![image](https://user-images.githubusercontent.com/5114450/119436474-6b455b80-bd46-11eb-8865-8b7f30826a8d.png)

Reply:
I played with indentations and found a way to make GitHub diff happier. Now, lines like `? UPPERCASE().union(LOWERCASE(), TITLECASE())` are probably a little bit too far to the right but it's still acceptable and diff looks much easier to review now.